sbtc-core/src/signer/mod.rs

```diff
@@ -125,3 +132,22 @@ impl<S: Sign + Coordinate> Keys for Signer<S> {
         todo!()
     }
 }
+
+impl<S: Sign + Coordinate> Validator for Signer<S> {
```

Comment:
I assume we'll need `Reveal` here too?
```suggestion
impl<S: Sign + Coordinate + Reveal> Validator for Signer<S> {
```